Gems/Atom/RPI/Code/Source/RPI.Public/Material/Material.cpp

```diff
@@ -280,7 +242,7 @@ namespace AZ
             // Note that it might not be strictly necessary to reinitialize the entire material, we might be able to get away with
             // just bumping the m_currentChangeId or some other minor updates. But it's pretty hard to know what exactly needs to be
             // updated to correctly handle the reload, so it's safer to just reinitialize the whole material.
-            OnAssetReloaded(m_materialAsset);
+            Init(*m_materialAsset);
```

Comment:
Hm, it's interesting that you were able to remove the material reload notification bus, but this is still here. Are you planning to remove this as well?
Though I expect it won't work until this is fixed: https://github.com/o3de/o3de/issues/13188